src/main/webapp/i18n/en/textExercise.json

```diff
@@ -29,7 +29,17 @@
             "checkPlagiarism": "Check Plagiarism",
             "exampleSubmissionsRequireExercise": "To create an example submission, you first need to finish creating the text exercise. You will be redirected automatically.",
             "saveSuccessful": "Your answer was saved successfully!",
-            "error": "An error occurred. Please try again later."
+            "error": "An error occurred. Please try again later.",
+            "clusterStatistics": {
+                "title": "Cluster Statistics",
+                "clusterId": "Cluster ID",
+                "reusedFeedbackRatio": "Cluster Reused Feedback Ratio",
+                "reusedFeedbackRatioTooltip": "Number of Automatic Feedbacks Reused / Cluster Size",
```

Comment:
As above, I am pretty sure, that there is no plural of "Feedback" (neither in German nor in English)
I would suggest: "Ratio of Reused Feedback in Cluster", "Amount of reused automatic feedback / cluster size"